tests/models/test_transformers.py

```diff
@@ -153,3 +153,34 @@ def test_phi3_chat_fixed(phi3_model: models.Model):
         lm += gen(name="five", max_tokens=10)
 
     assert "5" in lm["five"]
+
+
+def test_phi3_newline_chat():
```

Comment:
Can you just get the `phi3_model` fixture for lm, like `test_phi3_chat_fixed()` above?